java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/htmlui/Buttons.java

```diff
@@ -29,16 +29,15 @@ private Buttons() {
     }
 
     @JavaScriptBody(args = {}, body = "\n"
-        + "const vscode = acquireVsCodeApi();\n" // this method can be called only once per WebView existance
         + "window.close = function() {\n"
         + "  vscode.postMessage({\n"
```

Comment:
Another usage of `vscode` - one would need to capture it into a local variable and then delete it:
```
(function() {
  var vscode = globalThis.vscode;
  delete globalThis.vscode;
  
  globalThis.toBrwsrSrvr = ....
)();
```